crates/sb_workers/user_workers.js

```diff
@@ -49,7 +50,9 @@ class UserWorker {
 
         const res = await core.opAsync("op_user_worker_fetch_send", this.key, requestRid);
         const bodyStream = readableStreamForRid(res.bodyRid);
-        return new Response(res.size ? bodyStream : null, {
+        const uint8ResBody = (await readableStreamCollectIntoUint8Array(bodyStream)).buffer;
```

Comment:
Wouldn't this now buffer any streamed responses? It seems like that would make it impossible to use this to develop anything where streaming of a response is required. 

Streaming worked fine before the 204 fix so maybe there's a way to preserve both things?

Reply:
In the long run we want to be able to do this. I'd consider this a sort of quick fix while we figure some stuff out with Streamers, and the way the responses are shared through channels. We've discussed this, and even with Deno, there's cases where ReadableStream is also consumed internally when being returned. There's some challenges around this and it'll get to our list of priorities :)